### PR TITLE
Sync Lab Updates

### PR DESCRIPTION
**Pull request by Kenneth-LJS**
Changes:

image

1. Copy, apply format, set formats and delete format icons have been added to the UI. #1276
image
From left to right: Paste, Edit, Delete
1. Hovering over the icons will display a tooltip indicating what the button does. #1278
image
image
image
image

1. Copy button now looks responsive when clicking
image
1. Items can be deleted by clicking the delete button with the trash icon. #1273
1. New formats added to the list will be selected when copying. #1268
1. Formats are saved into a different powerpoint file, and are hence unaffected by undo. #1274
1. Un-copyable items display an error message when trying to copy them #1275
1. Display images will now use a separate template to save, hence hiding the SyncLab design template away from the user (we don't want them to touch this). #1288
1. Templates will be saved on attempting to exit. #1287
1. Fixed undo entries (broke when 2 separate PRs were merged)

Icons used are from Google's open source Material Icons set.

### CODE REVIEW

**Comment by damithc** [comment]
Be careful when using icons that already has meaning associated with them. The first one looks like `download` and the second one looks like `settings`

**Comment by Kenneth-LJS** [comment]
This looks the most appropriate icon so far. The typical copy/cut/paste icons can be used if you like.

image

I avoided using them because the clipboard by itself might not look intuitive.

File: PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs.SyncLab
+{
+    public class SyncLabShapeStorage : PowerPointPresentation
+    {
+#pragma warning disable 0618
+
+        int nextKey = 0;
```
Comment (jamos-tay) [inline PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs]: Access modifier

**Comment by damithc** [comment]
Yes, clipboard is not very intuitive for our case. May be the one you have is OK for paste.

**Pull request merged**